### PR TITLE
Implement serialize and deserialize for WGS84

### PR DESCRIPTION
**Pull request by meltinglava**
Points of discussion:
1. Do we want to serialize as degrees or radians?
2. Any way of testing this neatly?

### CODE REVIEW

**Comment by meltinglava** [comment]
Fixes #5

**Comment by feuerste** [comment]
Did you try https://docs.serde.rs/serde_test/ for testing?

**Comment by meltinglava** [comment]
Testing was left out, more to have creating some more realistic examples, as well as actually commiting to useing radians. I am still thinking that we might want to use degrees, as most navigational tools use them.

**Comment by meltinglava** [comment]
After some work i have desided that degrees are the way to go. serde_test has a fault that makes it fail as convertion to and from radians changes the float slightly. Have posted an issue in serde.

**Comment by feuerste** [comment]
I don't agree that (de)serialization should change the unit of how angles are stored internally. This leads exactly to the precision problems you describe. We need to stay with radians.

**Comment by meltinglava** [comment]
I agree that precision is better than degrees. However as i have coded it now serde is an optional dependency. Do we want to have it in default (opt in or opt out).

In case of not in default ill make some changes to CI to run the tests with `--all-features`

**Comment by feuerste** [comment]
It should be optional.

**Comment by meltinglava** [comment]
Think i am done now. Thow we might want to validate that the float is in range when we deserialize the values.

File: .travis.yml [hunk 1]
```diff
@@ -6,3 +6,7 @@ rust:
 matrix:
   allow_failures:
     - rust: nightly
+script:
+  - cargo build --no-default-features
```
Comment (feuerste) [inline .travis.yml]: How about also enabling `--verbose` here?

Comment (feuerste) [reply .travis.yml]: Do we need to test this at all?

Comment (meltinglava) [reply .travis.yml]: I agree with the `--verbose` flag. And I would say yes. There might be some forgotten `#[cfg(feature="<<flag>>"]` missing that makes it not compile

File: .travis.yml [hunk 2]
```diff
@@ -6,3 +6,7 @@ rust:
 matrix:
   allow_failures:
     - rust: nightly
+script:
+  - cargo build  --verbose --no-default-features
```
Comment (feuerste) [inline .travis.yml]: Nit: Remove one whitespace.

File: Cargo.toml [hunk 3]
```diff
@@ -14,12 +14,15 @@ license = "MIT"
 
 [dependencies]
 nalgebra = "0.21"
+serde = {version = "1.0.105", optional = true, features = ["serde_derive"]}
```
Comment (feuerste) [inline Cargo.toml]: Nit: Add whitespace behind `{` and before `}`

**Pull request merged**